src/jobs/release-package.yml

```diff
@@ -155,3 +168,25 @@ steps:
       condition:
         not: <<parameters.dryrun>>
       steps: <<parameters.post-job-steps>>
+
+  # clean up the CTC cases
+  - when:
+      condition:
+        equal: ['latest', <<parameters.tag>>]
+      steps:
+        - run:
+            when: on_fail
+            command: |
+              if [ -z "${SF_CHANGE_CASE_SFDX_AUTH_URL}" ] || [ -z "${SF_CHANGE_CASE_TEMPLATE_ID}" ] || [ -z "${SF_CHANGE_CASE_SCHEDULE_BUILD}" ]; then
+                  echo "Environment not configured for CTC"
+              else
+                  sfchangecase close --location "https://github.com/$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME" --status "Not Implemented"
+              fi
+        - run:
+            when: on_success
+            command: |
+              if [ -z "${SF_CHANGE_CASE_SFDX_AUTH_URL}" ] || [ -z "${SF_CHANGE_CASE_TEMPLATE_ID}" ] || [ -z "${SF_CHANGE_CASE_SCHEDULE_BUILD}" ]; then
+                  echo "Environment not configured for CTC"
+              else
+                  sfchangecase close --location "https://github.com/$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME"
```

Comment:
Can these be closed without a specified status?

Reply:
the changecase command has a default.  You only need a status if it didn't go well